ReactNativeClient/lib/joplin-renderer/MdToHtml/rules/copy_to_clipboard.js

```diff
@@ -0,0 +1,108 @@
+const fs = require('fs');
+const path = require('path');
+const filePath = path.join(__dirname, '../copyFunction.js');
+const btnCode = fs.readFileSync(filePath).toString();
```

Comment:
That's an interesting approach to load the JS, but that won't work here because the "fs" package or reading file from the filesystem won't work on mobile (or browser, where this package can also be used). I think you can simply bring bring the copyFunction in this file directly.

Reply:
done